BrowserKit/Sources/ComponentLibrary/ActionButton.swift

```diff
@@ -35,8 +35,7 @@ public class ActionButton: ResizableButton {
         updatedConfiguration.title = viewModel.title
         updatedConfiguration.titleTextAttributesTransformer = UIConfigurationTextAttributesTransformer { incoming in
             var outgoing = incoming
-            outgoing.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .subheadline,
-                                                                   size: viewModel.fontSize)
+            outgoing.font = FXFontStyles.Regular.subheadline.scaledFont().withSize(viewModel.fontSize)
```

Comment:
@bmihai23 Is there anywhere where we are using a different `viewModel.fontSize` from `15`? If not, we can just use `FXFontStyles.Regular.subheadline.scaledFont()` for simplicity and remove the `fontSize` property.